README.md

```diff
@@ -2,27 +2,24 @@
 
 Repo containing the initLab's IVR telephony system, which enables opening of doors.
 
-This was codenamed `gosho-ivr` due to being able to open the initlab doors.
-
-
 ## Install
 
 
 ```
-git clone https://github.com/initlab/initlab-telephony /var/lib/asterisk/gosho-ivr
+git clone https://github.com/initlab/initlab-telephony /var/lib/asterisk/initlab-telephony
 
-cd /var/lib/asterisk/gosho-ivr
+cd /var/lib/asterisk/initlab-telephony
 python -m vevn .venv
 
 . .venv/bin/activate
 # pip install -r requirements.txt  # FIXME: pyst2 needs patches
 
-cd /var/lib/asterisk/gosho-ivr/sounds/; ./generate-messages.sh
+cd /var/lib/asterisk/initlab-telephony/sounds/; ./generate-messages.sh
 
 cp door_ivr/door_ivr.example.conf door_ivr/door_ivr.conf
 # edit door_ivr/door_ivr.conf
 
-for x in /var/lib/asterisk/gosho-ivr/sounds/files/*; do ln -s "$x" /usr/share/asterisk/sounds/en_US_f_Allison/; done
+for x in /var/lib/asterisk/initlab-telephony/sounds/files/*; do ln -s "$x" /usr/share/asterisk/sounds/en_US_f_Allison/; done
```

Comment:
@user890104 , keep in mind that if you want to move the folder you will need to:
- move the current solution
- move the virtualenv (`.venv`)
- change the asterisk conf

Reply:
I'll keep that in mind.